src/tiledb/cloud/soma/ingest.py

```diff
@@ -303,6 +360,19 @@ def run_ingest_workflow(
             )
         )
 
+    try:
+        ns, dst = utils.split_uri(output_uri)
+        namespace = namespace or ns
+        # Ensure compatibility with "tiledb://<namespace>/<bucket-path>"
+        # style URIs
+        if "://" in dst:
```

Comment:
@JohnMoutafis could we check the scheme of the output_uri instead? Like `if dst.startswith("tiledb")`? I think that would be more meaningful.

Reply:
The [`tiledb.cloud._common.utils.split_uri`](https://github.com/TileDB-Inc/TileDB-Cloud-Py/blob/5d4cd8207c34bc91b96253bd2736146088034a1d/src/tiledb/cloud/_common/utils.py#L90) already checks for that and throws a `ValueError` if the URI doesn't start with `tiledb://`.
That case is handled under the assumption that if the exception was thrown, a bucket URI was passed instead.

Reply:
Thanks for the explanation, @JohnMoutafis !